### PR TITLE
(1CT) Add Edit Param View

### PR DESCRIPTION
**Pull request by JoseRFelix**
## What is the purpose of the change

https://github.com/osmosis-labs/osmosis-frontend/assets/21092519/2eed8c31-7217-43cf-8120-5a7624d2469f

### ClickUp Task

[ClickUp Task URL](https://app.clickup.com/t/86a21b468)

## Brief Changelog

- Add Screen Manager Component to handle complex inner navigation between components
- Add edit 1CT params view
- Handle navigation between different 1CT settings.

## Testing and Verifying

- [ ] Should display 1CT edit params settings when clicking 'change' in wallet select modal 
- [ ] Should be able to click on a setting and navigate to its view

### CODE REVIEW

**Review by DavideSegullo — APPROVED** [review]

File: packages/web/components/one-click-trading/one-click-trading-settings.tsx [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+                    width={32}
+                    height={32}
+                    className="self-start"
+                  />
+                  <p className="text-body2 font-body2 text-osmoverse-300">
+                    {t("oneClickTrading.settings.description")}{" "}
+                    <a
+                      className={buttonCVA({
+                        mode: "text",
+                        className: "!inline w-auto px-0 text-body2 font-body2",
```
Comment (MaxMillington) [inline packages/web/components/one-click-trading/one-click-trading-settings.tsx:85]: Why the `!`?

File: packages/web/components/screen-manager.tsx [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+  );
+};
+
+interface ScreenProps {
+  screenName: string;
+  children: React.ReactNode | ((arg: ScreenManagerState) => React.ReactNode);
+}
+
+/**
+ * `ScreenManager` provides a context for managing screens within a React application,
```
Comment (MaxMillington) [inline packages/web/components/screen-manager.tsx:97]: didn't you include this note already above?

File: packages/web/components/one-click-trading/one-click-trading-settings.tsx (hunk 1 above)
Comment (JoseRFelix) [reply packages/web/components/one-click-trading/one-click-trading-settings.tsx:85]: We need it to override the default `display` of ButtonCVA

File: packages/web/components/screen-manager.tsx (hunk 2 above)
Comment (JoseRFelix) [reply packages/web/components/screen-manager.tsx]: Fixed on my latest commit. Thank you for the review!

**Pull request merged**